resources/definitions/fokoos_base.def.json

```diff
@@ -0,0 +1,215 @@
+            "ultimaker_pva",
+            "ultimaker_tough_pla_black",
+            "ultimaker_tough_pla_green",
+            "ultimaker_tough_pla_red",
+            "ultimaker_tough_pla_white",
+            "ultimaker_tpu_black",
+            "ultimaker_tpu_blue",
+            "ultimaker_tpu_red",
+            "ultimaker_tpu_white",
+            "volumic_arma_ultra",
+            "volumic_asa_ultra",
+            "volumic_br80_ultra",
+            "volumic_bumper_ultra",
+            "volumic_cu80_ultra",
+            "volumic_flex93_ultra",
+            "volumic_medical_ultra",
+            "volumic_nylon_ultra",
+            "volumic_pekk_carbone",
+            "volumic_petgcarbone_ultra",
+            "volumic_pp_ultra",
+            "volumic_strong_ultra",
+            "volumic_support_ultra",
+            "xyzprinting_carbon_fiber",
+            "xyzprinting_flexible",
+            "xyzprinting_nylon",
+            "xyzprinting_tough_pla",            
+            "zyyx_pro_flex"
+        ]      
+        
+    },
+    "overrides": {
+        "machine_name":        {"default_value": "FOKOOS Base Printer"},
+        "machine_start_gcode": {"default_value": "; Fokoos Custom Start G-code\nG92 E0 ; Reset Extruder\nG28 ; Home all axes\nG1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed\nG1 X0.1 Y20 Z0.3 F5000.0 ; Move to start position\nG1 X0.1 Y200.0 Z0.3 F1500.0 E15 ; Draw the first line\nG1 X0.4 Y200.0 Z0.3 F5000.0 ; Move to side a little\nG1 X0.4 Y20 Z0.3 F1500.0 E30 ; Draw the second line\nG92 E0 ; Reset Extruder\nG1 Z2.0 F3000 ; Move Z Axis up little to prevent scratching of Heat Bed\nG1 X5 Y20 Z0.3 F5000.0 ; Move over to prevent blob squish"},
+        "machine_end_gcode": { "default_value": "G91 ;Relative positioning\nG1 E-2 F2700 ;Retract a bit\nG1 E-2 Z0.2 F2400 ;Retract and raise Z\nG1 X5 Y5 F3000 ;Wipe out\nG1 Z10 ;Raise Z more\nG90 ;Absolute positioning\nG1 X0 Y{machine_depth} ;Present print\nM106 S0 ;Turn-off fan\nM104 S0 ;Turn-off hotend\nM140 S0 ;Turn-off bed\nM84 X Y E ;Disable all steppers but Z\n" },
+        
+        "machine_max_feedrate_x": { "value": 500 },
+        "machine_max_feedrate_y": { "value": 500 },
+        "machine_max_feedrate_z": { "value": 10 },
+        "machine_max_feedrate_e": { "value": 60 },
+
+        "machine_max_acceleration_x": { "value": 500 },
+        "machine_max_acceleration_y": { "value": 500 },
+        "machine_max_acceleration_z": { "value": 100 },
+        "machine_max_acceleration_e": { "value": 5000 },
+        "machine_acceleration": { "value": 500 },
+
+        "machine_max_jerk_xy": { "value": 10 },
+        "machine_max_jerk_z": { "value": 0.4 },
+        "machine_max_jerk_e": { "value": 5 },
+
+        "machine_heated_bed": { "default_value": true },
+        "material_diameter": { "default_value": 1.75 },
+
+
+        "line_width": { "value": "machine_nozzle_size" },
+
+
+        "wall_thickness": {"value": "line_width * 2" },
```

Comment:
This means that spiralize will not work for your printer. The default value is 
`"value": "wall_line_width_0 if magic_spiralize else 0.8",`